{"name": "test_login_2_success_login", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestLogin' object has no attribute 'driver'", "trace": "self = <TestCases.test_login.TestLogin object at 0x000001D73E058A90>\n\n    def test_login_2_success_login(self):\n        # 步骤 - 登陆操作 - 登陆页面 - 18684720553、python\n>       LoginPage(self.driver).login(ld.success_data[\"user\"], ld.success_data[\"passwd\"])  # 测试对象+测试数据\nE       AttributeError: 'TestLogin' object has no attribute 'driver'\n\nTestCases\\test_login.py:28: AttributeError"}, "start": 1562164538579, "stop": 1562164538579, "uuid": "5dc5c0a8-a736-4cb6-b3f7-2c090bffe6e6", "historyId": "70890bea4db005493b2ff2b6079d770f", "fullName": "TestCases.test_login.TestLogin#test_login_2_success_login", "labels": [{"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-F99FSHQ"}, {"name": "thread", "value": "5524-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_login"}]}